packages/reports/src/Constants/__tests__/data-table-constants.spec.tsx

```diff
@@ -0,0 +1,570 @@
+        default: ({ currency }: { currency: string }) => <div>{currency}</div>,
+    };
+});
+
+jest.mock('Components/market-symbol-icon-row', () => {
+    return {
+        __esModule: true,
+        default: ({ payload }: { payload: { id: string; transaction_id?: string } }) => (
+            <div>{payload.id || payload.transaction_id}</div>
+        ),
+    };
+});
+
+jest.mock('@deriv/components', () => ({
+    __esModule: true,
+    ContractCard: {
+        MultiplierCloseActions: ({ contract_info }: { contract_info: { contract_id: string } }) => (
+            <div>{contract_info.contract_id}</div>
+        ),
+    },
+    ContractCardSell: ({ contract_info }: { contract_info: { contract_id: string } }) => (
+        <div>{contract_info.contract_id}</div>
+    ),
+    Money: ({ amount }: { amount: string }) => <div>{amount}</div>,
+    ArrowIndicator: jest.fn(() => 'arrowIndicator'),
+}));
+
+describe('getStatementTableColumnsTemplate', () => {
+    const currency = 'USD';
+    const columns = getStatementTableColumnsTemplate(currency);
+
+    it('should return the correct number of columns', () => {
+        expect(columns).toHaveLength(7);
+    });
+    it('should correctly define the "Type" column for non-mobile', () => {
+        (isMobile as jest.Mock).mockReturnValue(false);
+        const typeColumn = columns[0];
+        expect(typeColumn.title).toBe('Type');
+    });
+    it('should correctly define the "Ref. ID" column', () => {
+        const refIdColumn = columns[1];
+        expect(refIdColumn.title).toBe('Ref. ID');
+    });
+    it('should correctly define the "Currency" column', () => {
+        const currencyColumn = columns[2];
+        expect(currencyColumn.title).toBe('Currency');
+    });
+    it('should correctly define the "Transaction time" column', () => {
+        const transactionTimeColumn = columns[3];
+        expect(transactionTimeColumn.title).toBe('Transaction time');
+    });
+    it('should correctly define the "Transaction" column', () => {
+        const transactionColumn = columns[4];
+        expect(transactionColumn.title).toBe('Transaction');
+    });
+    it('should correctly define the "Credit/Debit" column', () => {
+        const creditDebitColumn = columns[5];
+        expect(creditDebitColumn.title).toBe('Credit/Debit');
+    });
+    it('should correctly define the "Balance" column', () => {
+        const balanceColumn = columns[6];
+        expect(balanceColumn.title).toBe('Balance');
+    });
+    it('should render cell content for "Currency" column', () => {
+        const currencyColumn = columns[2];
+        render(currencyColumn.renderCellContent({} as TCellContentProps));
+
+        expect(screen.getByText(currency)).toBeInTheDocument();
+    });
+    it('should render cell content for "Transaction time" column', () => {
+        const transactionTimeColumn = columns[3];
+        const cell_value = '2023-01-01 12:00:00';
+        //@ts-expect-error type for cell_value is wrong
```

Comment:
Maybe we should create another card as a part if Tech debt to improve type in _packages/reports/src/Constants/data-table-constants.tsx_? Apart of your card.
Because type `cell_value: "default" | "adjustment" | "transfer" | "buy" | "deposit" | "hold" | "release" | "sell" | "withdrawal"` can't be used in all cases

Reply:
Same with other //@ts-expect-error

Reply:
sure